test/test_prototype_transforms_consistency.py

```diff
@@ -655,37 +655,58 @@ def test_call_consistency(config, args_kwargs):
     )
 
 
-@pytest.mark.parametrize(
-    "config",
-    [config for config in CONSISTENCY_CONFIGS if hasattr(config.legacy_cls, "get_params")],
-    ids=lambda config: config.legacy_cls.__name__,
+get_paramsl_parametrization = pytest.mark.parametrize(
+    ("config", "get_params_args_kwargs"),
+    [
+        pytest.param(
+            next(config for config in CONSISTENCY_CONFIGS if config.prototype_cls is transform_cls),
+            get_params_args_kwargs,
+            id=transform_cls.__name__,
+        )
+        for transform_cls, get_params_args_kwargs in [
+            (prototype_transforms.RandomResizedCrop, ArgsKwargs(make_image(), scale=[0.3, 0.7], ratio=[0.5, 1.5])),
```

Comment:
Do we need to use the ArgsKwargs class at all here?

Reply:
Define "need". You can also replace it with a sequence and a mapping, but I feel this is more convenient, since it closely matches how you would actually call the method. Plus, this is consistent with this module as a whole, which uses `ArgsKwargs` every time we need to define call arguments.